packages/shared-ux/chrome/navigation/src/ui/navigation.tsx

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import {
+  EuiCollapsibleNavGroup,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHeaderLogo,
+  EuiLink,
+  EuiLoadingSpinner,
+  EuiSpacer,
+  useEuiTheme,
+} from '@elastic/eui';
+import React from 'react';
+import { NavigationBucketProps, NavigationProps } from '../../types';
+import { NavigationModel } from '../model';
+import { useNavigation } from '../services';
+import { ElasticMark } from './elastic_mark';
+import './header_logo.scss';
+import { NavigationBucket } from './navigation_bucket';
+
+export const Navigation = (props: NavigationProps) => {
+  const { loadingCount, activeNavItemId, ...services } = useNavigation();
+  const { euiTheme } = useEuiTheme();
+
+  const activeNav = activeNavItemId ?? props.activeNavItemId;
+
+  const nav = new NavigationModel(services, props.platformConfig, props.solutions, activeNav);
+
+  const solutions = nav.getSolutions();
+  const { analytics, ml, devTools, management } = nav.getPlatform();
+
+  const NavHeader = () => {
+    const homeUrl = services.basePath.prepend(props.homeHref);
+    const navigateHome = (event: React.MouseEvent) => {
+      event.preventDefault();
+      services.navigateToUrl(homeUrl);
+    };
+    const logo =
+      loadingCount === 0 ? (
+        <EuiHeaderLogo
+          iconType="logoElastic"
+          aria-label="Go to home page"
```

Comment:
Maybe should be translated.

Reply:
99af20a7188282e547f712561125368dc9dbc4da